tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I'd remove `noImplicitAny`, `noImplicitThis`, and `strictNullChecks`, replacing them all with just `strict`, which implies those three and more. But also note that this tsconfig probably isn't even necessary unless there are tests committed to this repo; I don't think you'll need a tsconfig present for someone else to require your types.

Reply:
Since we're including `*.ts` tests, I think I'll go the `strict` route, but happy to modify as needed.